{"kind":"symbol","sections":[],"schemaVersion":{"major":0,"patch":0,"minor":3},"hierarchy":{"paths":[["doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit","doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute"]]},"primaryContentSections":[{"declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"system","kind":"identifier"}]}],"kind":"declarations"}],"abstract":[{"text":"Error caused by a system that the process runs on.","type":"text"},{"text":" ","type":"text"},{"type":"text","text":"Hardware failure, OS-level error, resource exhaustion,"},{"type":"text","text":" "},{"text":"environment configuration, …","type":"text"}],"metadata":{"roleHeading":"Case","title":"SourceErrorAttribute.system","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"system"}],"symbolKind":"case","modules":[{"name":"SwiftErrorKit"}],"role":"symbol","externalID":"s:13SwiftErrorKit06SourceB9AttributeO6systemyA2CmF"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swifterrorkit\/sourceerrorattribute\/system"]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute\/system"},"references":{"doc://SwiftErrorKit/documentation/SwiftErrorKit/SourceErrorAttribute":{"url":"\/documentation\/swifterrorkit\/sourceerrorattribute","abstract":[{"type":"text","text":"An error attribute to include information about the high-level component"},{"type":"text","text":" "},{"text":"in which the error emerged.","type":"text"}],"navigatorTitle":[{"kind":"identifier","text":"SourceErrorAttribute"}],"type":"topic","kind":"symbol","identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"SourceErrorAttribute"}],"title":"SourceErrorAttribute","role":"symbol"},"doc://SwiftErrorKit/documentation/SwiftErrorKit/SourceErrorAttribute/system":{"url":"\/documentation\/swifterrorkit\/sourceerrorattribute\/system","role":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"system","kind":"identifier"}],"title":"SourceErrorAttribute.system","identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/SourceErrorAttribute\/system","kind":"symbol","type":"topic","abstract":[{"type":"text","text":"Error caused by a system that the process runs on."},{"type":"text","text":" "},{"type":"text","text":"Hardware failure, OS-level error, resource exhaustion,"},{"type":"text","text":" "},{"type":"text","text":"environment configuration, …"}]},"doc://SwiftErrorKit/documentation/SwiftErrorKit":{"title":"SwiftErrorKit","url":"\/documentation\/swifterrorkit","type":"topic","identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit","kind":"symbol","abstract":[{"text":"Error attribution and commons.","type":"text"}],"role":"collection"}}}